index.js

```diff
@@ -13,18 +14,32 @@
 // *** otherwise, the error will stop the rest of your code from running
 // console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
+const name = "Raffy"
+console.log(name)
 // => "Raffy"
-// name = "Not Raffy" 
+name = "Not Raffy" 
 // => TypeError
 // console.log("%c----------", "color: red") 
 
 
-
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 3", "color: red") 
 
+let currentThirstLevel = 12
+
+function drinkWater(currentThirstLevel) {
+  let currentThirstLevel = 12; 
+  while (currentThirstLevel > 11) 
+     return "Man I sure am thirsty";
+     currentThirstLevel -= 1; 
```

Comment:
- There's no opening curly brace to open the block for the while loop. That will cause an error. Be sure to test your code since the curly braces in the code will not match up
- I want to point out that any code written after a `return` statement will not run. When JavaScript sees the `return` it exists the function
- However, you don't need a while loop for this solution (The Ruby method you're meant to translate from doesn't have a while loop).